tests/unit/plugins/service/test_gdns_publisher.py

```diff
@@ -371,34 +325,19 @@ def test_implements_interface(mocker, config, metrics):
 
 @pytest.mark.parametrize('exception_args,http_exception', http_exceptions)
 @pytest.mark.asyncio
-async def test__dispatch_changes_http_exceptions_raised(
+async def test_dispatch_changes_http_exceptions_raised(
```

Comment:
The second _ was deliberate.  The first one is a word separator, the second is the leading _ from the method.